### PR TITLE
New package: sslh-1.17

### PR DESCRIPTION
**Pull request by pulux**
New package: sslh-1.17

### CODE REVIEW

File: srcpkgs/sslh/template [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+do_install () {
+	make install DESTDIR="${DESTDIR}"
+}
+
+post_install() {
+	vman sslh.8
+	vsv sslh
+	for f in *.cfg; do
+		vsconf ${f}
+		vsconf README.md
```
Comment (beefcurtains) [inline srcpkgs/sslh/template]: You want to install this file twice? And why not ``vdoc``?

File: srcpkgs/sslh/template [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+do_build() {
+	make
+}
+
+do_install () {
+	make install DESTDIR="${DESTDIR}"
+}
+
+post_install() {
+	vman sslh.8
```
Comment (beefcurtains) [inline srcpkgs/sslh/template]: Could also gunzip here, instead of invoking sed on the Makefile.

File: srcpkgs/sslh/template [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+	sed -i 's|sbin|bin|g' Makefile
+	sed -i 's|sslh.pod \| gzip -9 - > $(MAN)|sslh.pod > $(MAN)|g' Makefile
+}
+
+do_build() {
+	make
+}
+
+do_install () {
+	make install DESTDIR="${DESTDIR}"
```
Comment (beefcurtains) [inline srcpkgs/sslh/template]: ``build_style=gnu-makefile``, it does this for you.

File: srcpkgs/sslh/template [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+pre_build() {
+	sed -i 's|^PREFIX=/usr/local|PREFIX=/usr|g' Makefile
+	sed -i 's|^MAN=sslh.8.gz|MAN=sslh.8|g' Makefile
+	sed -i 's|sbin|bin|g' Makefile
+	sed -i 's|sslh.pod \| gzip -9 - > $(MAN)|sslh.pod > $(MAN)|g' Makefile
+}
+
+do_build() {
+	make
```
Comment (beefcurtains) [inline srcpkgs/sslh/template]: ``build_style=gnu-makefile``, it does this for you.

File: srcpkgs/sslh/template [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+license="GPL-2"
+homepage="http://www.rutschle.net/tech/sslh"
+distfiles="https://github.com/yrutschle/sslh/archive/v${version}.tar.gz"
+checksum=b557c3bf75289d203353c25d92e34549a22f64510c48f7748c1c027748d51ad1
+
+
+pre_build() {
+	sed -i 's|^PREFIX=/usr/local|PREFIX=/usr|g' Makefile
+	sed -i 's|^MAN=sslh.8.gz|MAN=sslh.8|g' Makefile
+	sed -i 's|sbin|bin|g' Makefile
```
Comment (beefcurtains) [inline srcpkgs/sslh/template]: ``make_install_args="BINDIR=/usr/bin"`` if you must (it's not really needed anymore).

File: srcpkgs/sslh/template [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+short_desc="Applicative Protocol Multiplexer"
+maintainer="Markus Berger <pulux@pf4sh.de>"
+license="GPL-2"
+homepage="http://www.rutschle.net/tech/sslh"
+distfiles="https://github.com/yrutschle/sslh/archive/v${version}.tar.gz"
+checksum=b557c3bf75289d203353c25d92e34549a22f64510c48f7748c1c027748d51ad1
+
+
+pre_build() {
+	sed -i 's|^PREFIX=/usr/local|PREFIX=/usr|g' Makefile
```
Comment (beefcurtains) [inline srcpkgs/sslh/template]: Not needed with ``build_style=gnu-makefile``.

File: srcpkgs/sslh/template [hunk 7]
```diff
@@ -0,0 +1,37 @@
+# Template file for 'sslh'
+pkgname=sslh
+version=1.17
+revision=1
+makedepends="libconfig-devel libcap-devel perl"
+depends="libconfig"
```
Comment (beefcurtains) [inline srcpkgs/sslh/template]: Should be automatic.

File: srcpkgs/sslh/template [hunk 8]
```diff
@@ -0,0 +1,29 @@
+# Template file for 'sslh'
+pkgname=sslh
+version=1.17
+revision=1
+only_for_archs="i686 x86_64"
```
Comment (xtraeme) [inline srcpkgs/sslh/template]: This only_for_archs is completely wrong, please remove this.

**Comment by pulux** [comment]
`only_for_archs="i686 x86_64"`

generated `xnew`  from the `xtools` pkg.

**Pull request closed without merging**